fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchReverseChainTest.java

```diff
@@ -0,0 +1,1034 @@
+    public void testCreatePatient1() throws Exception {
+        WebTarget target = getWebTarget();
+
+        // Build a new Patient and then call the 'create' API.
+        Patient patient = TestUtil.getMinimalResource(ResourceType.PATIENT, Format.JSON);
+        patient = patient.toBuilder()
+                .gender(AdministrativeGender.MALE)
+                .name(HumanName.builder()
+                    .given(of("1" + tag))
+                    .build())
+                .meta(Meta.builder()
+                    .tag(Coding.builder()
+                        .code(Code.of(tag))
+                        .build())
+                    .build())
+                .generalPractitioner(Reference.builder().reference(of("PractitionerRole/" + tag)).build())
+                .managingOrganization(Reference.builder().reference(of("Organization/" + organization1Id)).build())
+                .birthDate(Date.of(now.toString().substring(0,10)))
+                .telecom(ContactPoint.builder().system(ContactPointSystem.PHONE).value(of("1" + tag)).build())
+               .build();
+
+        Entity<Patient> entity = Entity.entity(patient, FHIRMediaType.APPLICATION_FHIR_JSON);
+        Response response = target.path("Patient").request().post(entity, Response.class);
+        assertResponse(response, Response.Status.CREATED.getStatusCode());
+
+        // Get the patient's logical id value.
+        patient1Id = getLocationLogicalId(response);
+
+        // Next, call the 'read' API to retrieve the new patient and verify it.
+        response = target.path("Patient/" + patient1Id).request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
+        assertResponse(response, Response.Status.OK.getStatusCode());
+    }
+
+    @Test(groups = { "server-search-reverse-chain" }, dependsOnMethods = {"testCreatePatient1", "testCreateOrganization2"})
+    public void testCreatePatient2() throws Exception {
+        WebTarget target = getWebTarget();
+
+        // Build a new Patient and then call the 'create' API.
+        Patient patient = TestUtil.getMinimalResource(ResourceType.PATIENT, Format.JSON);
+        patient = patient.toBuilder()
+                .gender(AdministrativeGender.FEMALE)
+                .name(HumanName.builder()
+                    .given(of("2" + tag))
+                    .build())
+                .meta(Meta.builder()
+                    .tag(Coding.builder()
+                        .code(Code.of(tag))
+                        .build())
+                    .build())
+                .generalPractitioner(Reference.builder().reference(of("Practitioner/" + tag)).build())
+                .managingOrganization(Reference.builder().reference(of("Organization/" + organization2Id)).build())
+                .birthDate(Date.of(now.minus(1, ChronoUnit.DAYS).toString().substring(0,10)))
+                .telecom(ContactPoint.builder().system(ContactPointSystem.PHONE).value(of("2" + tag)).build())
+                .build();
```

Comment:
same formatter comment

Reply:
I'll leave you to decide to use it in the rest of the class (treat the formatter comments as FYI)